### PR TITLE
Capability of specifying shape for tiling images

### PR DESCRIPTION
**Pull request by wkentaro**
image

### CODE REVIEW

File: jsk_perception/node_scripts/tile_image.py [hunk 1]
```diff
@@ -70,15 +80,19 @@ def callback(msg):
     def _append_images(self, imgs):
         if not imgs:
             return
+        # convert tile shape: (Y, X) -> (X, Y)
+        shape_xy = self._shape[::-1] if self._shape else self._shape
```
Comment (furushchev) [inline jsk_perception/node_scripts/tile_image.py]: @wkentaro What do you mean about `else self._shape`? If you mean `None` here, I think using `None` is easier to understand.

File: doc/jsk_perception/nodes/tile_image.rst [hunk 2]
```diff
@@ -49,6 +49,11 @@ Parameters
 
   Font size to draw topic names.
 
+* ``shape`` (type: ``FloatArray``, default: ``None``)
+
+  Tile shape like ``[2, 4]`` in [Y, X] order.
+
```
Comment (furushchev) [inline doc/jsk_perception/nodes/tile_image.rst]: How does a shape goes if this param is `None`? Random?

**Comment by wkentaro** [comment]
I just rebased to origin/master.

**Comment by wkentaro** [comment]
@furushchev OK?

**Pull request merged**